index.html

```diff
@@ -5008,7 +5008,7 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
   </aside>
 
   <p>The example above would associate the <code>ja</code> language
-    code with the two <a>strings</a> <em>花澄</em> and <em>科学者</em>
+    code with the two <a>strings</a> <em lang="ja">花澄</em> and <em lang="ja">科学者</em>
     <a data-cite="BCP47#section-2">Languages codes</a> are defined in [[BCP47]].
```

Comment:
Nit: the term "language code" is generally associated with ISO 639. The term "language tag" (or subtag, in the case of elements of a tag) is usually used for BCP47 values.

Reply:
Fixed in 9b1e522554ac86b8079f21b10ca254a2c478ec2a.